test-monitor/nitestmonitor.yml

```diff
@@ -314,7 +314,7 @@ definitions:
         type: string
         example: admin
       product:
-        description: The type of device under test. Added in version 2.
+        description: The product of the device under test. Added in version 2.
```

Comment:
Looks like this file got merged in the wrong direction. According to https://github.com/ni/systemlink-OpenAPI-documents/commit/9de27516a1397f82f619817ff5c23953e66508aa "type" was correct.